SPECS/openssh.spec

```
@@ -653,10 +654,12 @@ getent passwd sshd >/dev/null || \
 %endif
 
 %changelog
-* Mon Apr 29 2024 Thierry Escande <thierry.escande@vates.tech> - 7.4p1-23.2.1 + 0.10.3-2
+* Tue Apr 30 2024 Thierry Escande <thierry.escande@vates.tech> - 7.4p1-23.2.1 + 0.10.3-2.23.2.1
```

Are you sure? I'd expect this:
```suggestion
* Tue Apr 30 2024 Thierry Escande <thierry.escande@vates.tech> - 7.4p1-23.2.1 + 0.10.3-2.1
```

---

The pam_ssh_agent release is defined as `Release: %{pam_ssh_agent_rel}.%{xsrel}.%{xcpng_subrel}%{?dist}`

---

Right, so it's the former changelog from XS which is wrong :thinking:

---

I've just added Release: `.%{xcpng_subrel}`. It was badly reported in previous changelogs